### PR TITLE
Update Virtualization descriptions

### PR DESCRIPTION
**Pull request by sradco**
Update Virtualization descriptions based on the docs team review.

Signed-off-by: Shirly Radco <sradco@redhat.com>

### CODE REVIEW

File: operators/multiclusterobservability/manifests/base/grafana/dash-acm-openshift-virtualization-overview.yaml [hunk 1]
```diff
@@ -100,7 +100,7 @@ data:
         },
         {
           "datasource": null,
-          "description": "Total number of clusters that have alerts that are firing with the operator_health_impact level: critical.",
+          "description": "Total number of clusters with critical issues that impact on the operator health, based on firing alerts and operator conditions. This means they lost some core functionality.",
```
Comment (dockerymick) [inline operators/multiclusterobservability/manifests/base/grafana/dash-acm-openshift-virtualization-overview.yaml:103]:
```suggestion
          "description": "Total number of clusters with critical issues that impact the operator health, due to alerts and operator conditions. This means they lost some core functionality.",
```

**Comment by moadz** [comment]
@dockerymick I'm not too worried about the titles because we would need to change the existing dashboards to follow the same format. This is fine for now. @sradco are you able to action the outstanding doc items apart from the titles and I can get this merged.

**Review by moadz — APPROVED** [review]
/lgtm

**Comment by dockerymick** [comment]
> @dockerymick I'm not too worried about the titles because we would need to change the existing dashboards to follow the same format. This is fine for now. @sradco are you able to action the outstanding doc items apart from the titles and I can get this merged.

So can the titles updates be made during a refresh or z-stream release? They really need to be improved

**Pull request merged**